lib/ui/views/settings/settingsFragment/settings_manage_api_url.dart

```diff
@@ -94,6 +94,7 @@ class SManageApiUrl extends BaseViewModel {
               _managerAPI.setApiUrl('');
               _toast.showBottom('settingsView.restartAppForChanges');
               Navigator.of(context).pop();
+              Navigator.of(context).pop();
```

Comment:
Use cascade notation for this function.

Reply:
Hello @TheAabedKhan, request changes are made please review it.